Python/Product/Cookiecutter/CookiecutterPackage.cs

```diff
@@ -89,6 +89,8 @@ public sealed class CookiecutterPackage : Package, IOleCommandTarget {
             _projectSystem = new ProjectSystemClient(DTE);
 
             CookiecutterTelemetry.Initialize();
+
+            VSTaskExtensions._telemetry = CookiecutterTelemetry.Current;
```

Comment:
What's the value of `_telemetry`? `CookiecutterTelemetry.Current` is accessible from `VSTaskExtensions`

Reply:
That was the reason, but might as well just use one global instead of two.